src/app/parcels/batch/BatchParcelDataGrid.tsx

```diff
@@ -34,200 +38,224 @@ export interface BatchGridDisplayRow {
+        },
+        {
+            field: "extraInformation",
+            headerName: "Extra Information",
+            width: 200,
+            editable: true,
+        },
+        {
+            field: "attentionFlag",
+            headerName: "Attention Flag",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "signpostingCall",
+            headerName: "Signposting Call",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "notes",
+            headerName: "Notes",
+            width: 200,
+            editable: true,
+        },
+        {
+            field: "voucherNumber",
+            headerName: "Voucher Number",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "packingDate",
+            headerName: "Packing Date",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "packingSlot",
+            headerName: "Packing Slot",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "shippingMethod",
+            headerName: "Shipping Method",
+            width: 150,
+            editable: true,
+        },
+        {
+            field: "collectionInfo",
+            headerName: "Collection Info",
+            width: 200,
+            editable: true,
+        },
+    ];
+    return batchGridDisplayColumns.map((column) => {
         return { ...column, headerAlign: "center", align: "center" };
     });
+};
 
 export const defaultTableState: BatchTableDataState = {
     overrideDataRow: {
-        data: null,
+        data: emptyOverrideData,
     },
     batchDataRows: [
         {
             id: 1,
             clientId: null,
-            data: null,
+            data: emptyBatchEditData,
         },
     ],
     clientOverrides: [],
     parcelOverrides: [],
 };
 
 const BatchParcelDataGrid: React.FC = () => {
-    const [tableState, _] = useLocalStorage(batchParcelsReducer, defaultTableState);
-
+    const [tableState, dispatch] = useLocalStorage(batchParcelsReducer, defaultTableState);
     const displayRows = useMemo(() => {
         return tableStateToBatchDisplayRows(tableState);
     }, [tableState]);
-
+    const theme: DefaultTheme = useTheme();
+    const centeredBatchGridDisplayColumns: GridColDef[] = getCenteredBatchGridDisplayColumns(
+        tableState,
+        dispatch
+    );
     return (
         <DataGrid
             rows={displayRows}
-            columns={styledBatchGridDisplayColumns}
+            columns={centeredBatchGridDisplayColumns}
             sx={{
                 "& .MuiDataGrid-cell": {
                     border: "1px solid",
-                    borderColor: "rgba(81, 81, 81, 0.5)",
+                    borderColor: `${theme.main.border}`,
+                    "& .MuiInputBase-root": {
+                        height: "100%",
+                    },
```

Comment:
Given that our usual pattern is styled components, might be worth leaving a small comment above the component explaining why we're doing it here.